ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CompletionStorageBackendTemplate.scala

```diff
@@ -76,4 +58,77 @@ trait CompletionStorageBackendTemplate extends CompletionStorageBackend {
         ORDER BY completion_offset ASC"""
       .as(completionParser.*)(connection)
   }
+
+  private val sharedColumns: RowParser[Offset ~ Instant ~ String ~ String ~ Option[String]] =
+    offset("completion_offset") ~
+      instant("record_time") ~
+      str("command_id") ~
+      str("application_id") ~
+      str("submission_id").?
+
+  private val acceptedCommandSharedColumns
+      : RowParser[Offset ~ Instant ~ String ~ String ~ Option[String] ~ String] =
+    sharedColumns ~ str("transaction_id")
+
+  private val deduplicationOffsetColumn: RowParser[Option[String]] = str("deduplication_offset").?
+  private val deduplicationTimeSecondsColumn: RowParser[Option[Long]] = long(
+    "deduplication_time_seconds"
+  ).?
+  private val deduplicationTimeNanosColumn: RowParser[Option[Int]] = int(
+    "deduplication_time_nanos"
+  ).?
+  private val deduplicationStartColumn: RowParser[Instant] = instant("deduplication_start")
+
+  private val acceptedCommandParser: RowParser[CompletionStreamResponse] =
+    acceptedCommandSharedColumns ~
+      deduplicationOffsetColumn ~
+      deduplicationTimeSecondsColumn ~ deduplicationTimeNanosColumn ~
+      deduplicationStartColumn map {
+        case offset ~ recordTime ~ commandId ~ _ ~ _ ~ transactionId ~
+            _ ~ _ ~ _ ~ _ =>
```

Comment:
These are the newly added (+ pre-existing application ID) command dedup columns, they will be used in the next PR.
```